src/main/kotlin/gg/essential/elementa/components/UIWrappedText.kt

```diff
@@ -19,9 +20,9 @@ import java.awt.Color
  * this component's width & height constrains.
  */
 open class UIWrappedText @JvmOverloads constructor(
-    text: String = "",
-    shadow: Boolean = true,
-    shadowColor: Color? = null,
+    text: State<String>,
+    shadow: State<Boolean>,
+    shadowColor: State<Color?>,
```

Comment:
I'd seem useful to provide defaults for these like the other constructor does (except for the first one, otherwise the no-arg call will be ambiguous).